packages/kbn-unified-field-list/src/containers/unified_field_list_sidebar/field_list_sidebar.scss

```diff
@@ -51,17 +51,12 @@
   align-items: center;
 }
 
+.unifiedFieldListItemButton__dragging {
+  @include euiSlightShadow;
+}
+
 .unifiedFieldListSidebar .unifiedFieldListItemButton {
   &.kbnFieldButton {
     margin-bottom: calc($euiSizeXS / 2);
   }
-
-  &.domDragDrop-isDraggable {
-    box-shadow: none;
-  }
-
-  &:not(.unifiedFieldListItemButton__dragging) {
-    padding: 0;
-    background: none;
-  }
```

Comment:
Thanks for the updates!

There is an issue now with an extra side padding in Discover sidebar:

<img width="217" alt="Screenshot 2024-03-12 at 10 46 23" src="https://github.com/elastic/kibana/assets/1415710/699a4ef7-df22-4d95-9efd-730692adb558">

And Lens items are missing shadow:
<img width="307" alt="Screenshot 2024-03-12 at 10 47 56" src="https://github.com/elastic/kibana/assets/1415710/687b02ad-60f5-4ead-9e8b-33430943621a">

Reply:
Thank you! I was checking on the dark theme and the shadow is not visible there 😅 Should be fine now, sorry!